### ISSUE TITLE
[ability draft] Search field clears spaces when typing slowly

### ISSUE THREAD

**Issue by p-fiftythree** [datdota/datdota#57]
The search field that appears on many pages (e.g. on https://windrun.io/abilities) removes leading and trailing spaces from the user's input after they've stopped typing for a short time. This can make it difficult to enter the names of abilities that contain spaces when unable to type quickly (e.g. one-handed on mobile).

An ideal solution would be to only trim spaces from the string passed to the search function, without modifying the user's input at all.

**Comment by Noxville** [comment]
This is just using the bootstrap table filter (https://getbootstrap.com/docs/4.0/content/tables/), not sure how much nicer it can be made out of the box -- last I checked there wasn't a very nice way to override the default behaviour. Let me know if you come across any ideas.

**Comment by p-fiftythree** [comment]
Yeah, looking at [the documentation](https://bootstrap-table.com/docs/api/table-options/#search) none of the options look good. A whole custom search mechanism doesn't seem worth it for such a minor gripe, so I'm happy to close this.
